email-templates/weekly/html.ejs

```diff
@@ -0,0 +1,579 @@
+                    var assetData = getAssetData(weekly.user.topAsset);
+                  -%>
+
+                    <table cellpadding="0" cellspacing="0">
+                      <tbody>
+                        <tr>
+                          <td class="preview-tile-container medium-container" style="background-image: url('<%= assetData.thumbnailUrl %>');" background="<%= assetData.thumbnailUrl %>" valign="bottom">
+                            <a href="<%= assetData.link %>" title="<%= assetData.title %>" target="_blank">
+                              <table cellpadding="0" cellspacing="0" class="preview-tile-container-inner medium-container-inner">
+                                <tr>
+                                  <td valign="bottom">
+                                    <div class="preview-tile-metadata">
+                                      <table cellpadding="0" cellspacing="0" class="preview-tile-metadata-inner">
+                                        <tr valign="top">
+                                          <td class="preview-tile-metadata-title">
+                                            <a href="<%= assetData.link %>" target="_blank" class="preview-tile-title wrapped"><%= assetData.title %></a>
+                                          </td>
+                                        </tr>
+                                        <tr>
+                                          <td class="preview-tile-metadata-description"><%= assetData.thumbnailSummary %></td>
+                                        </tr>
+                                      </table>
+                                    </div>
+                                  </td>
+                                </tr>
+                              </table>
+                            </a>
+                          </td>
+                        </tr>
+
+                        <tr>
+                          <td class="preview-asset-metadata">
+                            <img src="<%= baseCollabosphereUrl + resolveAsset('/assets/img/icon_thumbs_up.png') %>" alt="Number of likes" /> <%= weekly.user.topAsset.likes %>
+                            <img src="<%= baseCollabosphereUrl + resolveAsset('/assets/img/icon_views.png') %>" alt="Number of views" /> <%= weekly.user.topAsset.views %>
+                            <img src="<%= baseCollabosphereUrl + resolveAsset('/assets/img/icon_comments.png') %>" alt="Number of comments" /> <%= weekly.user.topAsset.comment_count %>
+                          </td>
+                        </tr>
+                      </tbody>
+                    </table>
+
+                  <% } -%>
+
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </td>
+        </tr>
+
+        <tr>
+          <td class="centered">
+            <div class="outbound-button">
+
+            <% var assetsUrl = course.assetlibrary_url + '?col_user=' + user.id; %>
```

Comment:
@paulkerschen should this be `col_user=' + weekly.user.id; %>`?  I don't see `user` declared.